src/Features/Core/Portable/ConvertToInterpolatedString/AbstractConvertPlaceholderToInterpolatedStringRefactoringProvider.cs

```diff
@@ -146,6 +146,27 @@ internal abstract class AbstractConvertPlaceholderToInterpolatedStringRefactorin
             return document.WithSyntaxRoot(newRoot);
         }
 
+        private string GetArgumentName(TArgumentSyntax argument, ISyntaxFactsService syntaxFactsService)
+            => syntaxFactsService.GetNameForArgument(argument);
+
+        private SyntaxNode GetParamsArgument(SeparatedSyntaxList<TArgumentSyntax> arguments, ISyntaxFactsService syntaxFactsService)
+        => arguments.FirstOrDefault(argument => string.Equals(GetArgumentName(argument, syntaxFactsService), StringFormatArguments.FormatArgumentName, StringComparison.OrdinalIgnoreCase)) ?? arguments[1];
+
+        private TArgumentSyntax GetFormatArgument(SeparatedSyntaxList<TArgumentSyntax> arguments, ISyntaxFactsService syntaxFactsService)
+            => arguments.FirstOrDefault(argument => string.Equals(GetArgumentName(argument, syntaxFactsService), StringFormatArguments.FormatArgumentName, StringComparison.OrdinalIgnoreCase)) ?? arguments[0];
```

Comment:
@jmarolf Since ```String.Format("{0}", ARg0: 1)``` is incorrect syntax 'convert to interpolate string' refactoring wouldn't be suggested